packages/loader/container-loader/src/quorum.ts

```diff
@@ -32,3 +34,26 @@ export class QuorumProxy extends EventForwarder<IQuorumEvents> implements IQuoru
         this.getMember = doIfNotDisposed(this, quorum.getMember.bind(quorum));
     }
 }
+
+export function getCodeProposalFromQuorumValues(
+    quorumValues: [string, ICommittedProposal][],
+): IFluidCodeDetails {
+    const qValuesMap = new Map(quorumValues);
+    const proposal = qValuesMap.get("code");
+    assert(proposal !== undefined, "Cannot find code proposal");
+    return proposal?.value as IFluidCodeDetails;
+}
+
+export function initQuorumCodeProposal(
```

Comment:
nit: `initQuorumValuesFromCodeDetails` to complement the name with the above function that does the opposite.